### DISCUSSION TITLE
Minibits from multiple devices?

### DISCUSSION THREAD

**Discussion by slcw** [minibits-cash/minibits_wallet#117] [Q&A]
Hey everyone... Is it possible to access my Minibits Wallet from multiple devices? I'd like to be able to install Minibits Wallet on a second device, and have it use my established minibits.cash address, and mints. Can this be done? Or is my Minibits.cash address and mint holdings dependant on a single device?

EDIT: 12/13/2024
I'm still curious about accessing my wallet from a second device. Is this possible? If I use my seed phrase and a current wallet backup, could I restore on a second device and use my funds from either device? I don't want to mess something up by trying, so I'll wait until I get a response on this.

**Comment by Marc26z** [comment]
I tried it a while ago. It worked back then.  Try it, but don't use this wallet for more sats than you are willing to use since it's very early days. Cashu is still in the experimental stage.

**Comment by minibits-cash** [comment]
For ecash to be recoverable, notes are generated by the wallet using incremental index counter.

When using same seed on two devices, those counters would conflict (if not assigned big enough, not overlapping intervals thus having still separate balances) leading to mint refuse the ecash notes (outputs already signed error).

If using the same nostr keys defining wallet address, another set of potential issues would arise with delivery of push notifications and of nostr messages.

Ecash is similar to your purse so that what's in does not exist elsewhere.
